fuzz/Cargo.toml

```diff
@@ -12,6 +12,12 @@ cargo-fuzz = true
 libfuzzer-sys = { version = "0.3" }
 fuzz_helper = { path = "fuzz_helper" }
 
+[build-dependencies]
+# We explicitly lock the version of those transitive dependencies because their
+# Cargo.toml don't parse with the nightly compiler used by Tock. Remove this
+# whole block once CTAP is a library.
+once_cell = { version = "=1.14", optional = true } # transitive dependency of ring
```

Comment:
This shouldn't be needed.

Reply:
I checked cargo tree, and once_cell is listed as a dependency of openssl. I noticed that I copy-pasted `ring` in the comment everywhere (about when you wrote your review), does that resolve why I had to add it to fuzz?

Reply:
The comment still applies, I don't see fuzzing depending on openssl.

Reply:
Ah right, it was just just transitive through the ctap module. And it failed for me before probably because I didn't cargo clean after testing something else.